apps/user-ms/src/main/docs/openapi.yaml

```diff
@@ -118,6 +118,38 @@ paths:
           description: Not Allowed
       security:
       - SecurityScheme: []
+  /institutions/{institutionId}/products/{productId}/admin-count:
```

Comment:
as api count the number of users it should be better using:
/institutions/{institutionId}/products/{productId}/users/count